From 6643feb187d3fd8e067dc3dfe707b583b75d312d Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 11 Jan 2024 16:43:24 +0100
Subject: [PATCH 0429/1695] plat-stm32mp2: Default enable
 CFG_DRIVERS_CLOCK_PRINT_TREE

Default enable CFG_DRIVERS_CLOCK_PRINT_TREE for stm32mp2 platform.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 214e8ee43b385ac029a2b2a448433ffbb6add329 plat-stm32mp2: Default enable CFG_DRIVERS_CLOCK_PRINT_TREE]
Change-Id: Ic94975313cf9bf2e495051bb70d9c0a65fe2c0e3
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index eb40b26a5..4d59139c9 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -96,6 +96,7 @@ CFG_REGULATOR_FIXED ?= y
 CFG_REGULATOR_GPIO ?= y
 
 # Default enable some test facitilites
+CFG_DRIVERS_CLK_PRINT_TREE ?= y
 CFG_DRIVERS_REGULATOR_PRINT_TREE ?= y
 CFG_ENABLE_EMBEDDED_TESTS ?= y
 CFG_WITH_STATS ?= y
-- 
2.47.2

